lib/cog/aggregate_command.rb

```diff
@@ -0,0 +1,57 @@
+require 'cog/exceptions'
+
+class Cog
+  class AggregateCommand < Cog::Command
+
+    def initialize
+      subcommands = self.class.const_get("SUBCOMMANDS")
+
+      unless subcommand = request.args.shift
+        no_subcommand_error
+        response.abort
+        return
```

Comment:
Only if `no_subcommand_error` or `unknown_subcommand_error` is overridden by the child class. I did it like that so I could have some control over error output in regards to missing args and whatnot.